lib/active_model/serializer.rb

```diff
@@ -199,16 +199,22 @@ def serializer_from_options(options)
       opts
     end
 
+    def self.serializers_cache
+      @serializers_cache ||= ThreadSafe::Cache.new
```

Comment:
btw it would generate a new dependency with ```thread_safe``` gem right?
Is there some reason to not use ```ActionController::Base.cache_store```